vindbaarheid/vindbaarheid.rdfs.ttl

```diff
@@ -0,0 +1,472 @@
+    rdfs:comment """A SKOS concept can be viewed as an idea or notion; a unit of thought. However, what constitutes a unit of thought is subjective, and this definition is meant to be suggestive, rather than restrictive."""@en .
+
+#####################################
+#             PROPERTIES            #
+#####################################
+
+#? Hoe literals declaren als range?
+
+####################
+#      PREMIS      #
+####################
+#? comments? Hoe differentiëren met schema properties?
+premis:hasPart a rdf:Property ;
+    rdfs:domain premis:IntellectualEntity ;
+    rdfs:range premis:IntellectualEntity ;
+    owl:equivalentProperty dct:hasPart ;
+    owl:inverseOf premis:isPartOf ;
+    rdfs:label "has part"@en ;
+    rdfs:comment ""@en .
+
+#? premis:identifier of schema:identifier?
+premis:identifier a rdf:Property ;
+    rdfs:domain premis:IntellectualEntity ;
+    rdfs:range meemoo:LocalIdentifier ;
+    rdfs:label "identifier"@en ;
+    rdfs:comment ""@en .
+
+premis:isPartOf a rdf:Property ;
+    rdfs:domain premis:IntellectualEntity ;
+    rdfs:range premis:IntellectualEntity ;
+    owl:equivalentProperty dct:isPartOf ;
+    owl:inverseOf premis:hasPart ;
+    rdfs:label "is part of"@en ;
+    rdfs:comment ""@en .
+
+premis:relationship a rdf:Property ;
+    rdfs:domain premis:IntellectualEntity ;
+    rdfs:range premis:IntellectualEntity ;
+    owl:equivalentProperty dct:relation ;
+    rdfs:label "relationship"@en ;
+    rdfs:comment ""@en .
+
+####################
+#      DCTERMS     #
+####################
+dct:available a rdf:Property ;
+    rdfs:domain premis:IntellectualEntity ;
+    rdfs:label "Date Available"@en ;
+    rdfs:comment "Date that the resource became or will become available."@en .
+
+dct:format a rdf:Property ;
+    #? dct:PhysicalResource?
+    rdfs:domain meemoo:PhysicalRepresentation ;
+    rdfs:label "format"@en ;
+    rdfs:comment "The file format, physical medium, or dimensions of the resource."@en .
+
+dct:medium a rdf:Property ;
+    rdfs:domain meemoo:PhysicalRepresentation ;
+    rdfs:label "medium"@en ;
+    rdfs:comment "The material or physical carrier of the resource"@en .
+
+####################
+#      EBUCORE     #
+####################
+ebu:objectType a rdf:Property #? rdf:Class? https://www.ebu.ch/metadata/ontologies/ebucore/ebucore_ObjectType.html
+    rdf:domain premis:IntellectualEntity ;
+    owl:equivalentProperty dct:type ;
+    rdfs:label "Object type"@en ;
+    rdfs:comment "To specify the type of BusinessObject e.g. and EditorialObject of type programme or clip"@en .
+
+####################
+#      SCHEMA      #
+####################
+schema:abstract a rdf:Property ;
+    rdfs:domain schema:CreativeWork ;
+    owl:equivalentProperty dct:abstract ; #?
+    rdfs:label "abstract"@en ;
+    rdfs:comment "An abstract is a short description that summarizes a CreativeWork."@en .
+    
+schema:actor a rdf:Property ;
+    rdfs:domain schema:PerformanceRole ;
```

Comment:
dit is een beetje een speciaal geval, want `schema:actor` kan ook het domain CreativeWork hebben. Je hebt constructies om dit uit te drukken, maar ik zou in dit geval gewoon CreativeWork nemen.

Reply:
Check ook de officiele definities: https://raw.githubusercontent.com/schemaorg/schemaorg/main/data/releases/14.0/schemaorg.owl. Het is altijd beter om lokale afwijkingen in de SHACL te steken